.lando.yml

```diff
@@ -1,10 +1,14 @@
 name: sous-project
-recipe: drupal8
+recipe: drupal9
```

Comment:
This is probably fine but since this is the Drupal 8 version maybe we keep the recipe with that version.

Reply:
@americkson I am testing backing the up to the Drupal 8 recipe but this does make some nice changes like upgrading composer to 2.0 and removing drush as a global install so that we can rely on the site install of drush.

Reply:
> upgrading composer to 2.0

This alone is worth it. Let's keep it.